config/dev.config.yaml

```diff
@@ -8,6 +8,7 @@ db:
 app:
   port: 3000
   network: main
+  batchSize: 100
```

Comment:
set it to something low. Maybe 5 or 3

Reply:
also because this option is specific to esplora, it should be a part of esplora config not under the app